velocity_vector: [0.7, 0.7, 1.2]
robot_name: "secondary_robot"
num_points_in_path: 100
robot_radius: 0.15
k_linear_wp: 1.5
k_linear_vp: 1.5
k : 0.02
min_dist_to_goal_point: 0.15
# max rate of the planner
RATE: 15
# goal tolerance
XY_GOAL_TOLERANCE: 0.005
YAW_GOAL_TOLERANCE: 0.01
# maximum speed
V_MAX: 0.7
W_MAX: 1.2
AV_MAX: 0.9
AW_MAX: 1.5
# acceleration at beginning of movements
ACCELERATION: 1.0
# goal distance for deceleration
D_DECELERATION: 0.15
R_DECELERATION: 1
exp_const: 0.01
# power of deceleration factor
GAMMA: 0.75
# params for accurate movements to water towers or cube heaps
XY_ACCURATE_GOAL_TOLERANCE: 0.005
YAW_ACCURATE_GOAL_TOLERANCE: 0.01
D_ACCURATE_DECELERATION: 0.20
collision:
    min_dist_to_obstacle_lidar: 0.5
    min_dist_to_obstacle_sensor: 0.2
    sensor_position: [[-0.131, 0., 3.14159265359], [0.06568, 0.11377, 1.0471975512], [0.06592, -0.11363, 5.21],
    [-0.04221, -0.0751, 4.19], [-0.02999, 0.08194, 2.08], [0.06634, -0.00163, 0.]]
    angle_sensor: 0.
